config/experiments/ccube/hyperparams_search_20230920_batch1.yaml

```diff
@@ -0,0 +1,145 @@
+# Shared config
```

Comment:
I don't mind much either way, but I wonder if it's actually important for all of these hyperparam configs to be merged.

Reply:
This could even be removed from the repo, so not important.

Reply:
I will remove them and keep them elsewhere ;)